src/xunit.runner.visualstudio/build/xunit.runner.visualstudio.desktop.props

```diff
@@ -1,11 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <EnableXunitRunner Condition=" '$(EnableXunitRunner)' == '' ">false</EnableXunitRunner>
+    <IsTestingPlatformApplication>$(EnableXunitRunner)</IsTestingPlatformApplication>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(EnableXunitRunner)' == 'true' ">
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- This GUID comes from Microsoft.Testing.Platform, do not modify -->
```

Comment:
The GUID is actually just any random GUID but it needs to be unique across the various extensions registered through the hook.